milli/src/update/index_documents/typed_chunk.rs

```diff
@@ -190,8 +158,20 @@ pub(crate) fn write_typed_chunk_into_index(
 
                 if !writer.is_empty() {
                     db.put(wtxn, &BEU32::new(docid), &writer.into_inner().unwrap())?;
+                    operations.push(DocumentOperation {
+                        external_id: external_id.to_string(),
+                        internal_id: docid,
+                        kind: DocumentOperationKind::Create,
+                    });
+                    docids.insert(docid);
```

Comment:
We now always push the external-internal docid pair when we create or update a document which is unnecessary for updates